addons/delete-others/userscript.js

```diff
@@ -0,0 +1,81 @@
+export default async function ({ addon, console, msg }) {
+  const vm = await addon.tab.traps.vm;
+
+  const types = ["sound", "costume"];
+  let deletedItems = [];
+  let deleted;
+  let target;
+
+  function getRestoreFun(type) {
+    if (type === "costume") {
+      deletedItems.forEach((costume) => {
+        vm.addCostume(costume.md5, costume, target.id, "");
+      });
+    } else if (type === "sound") {
+      deletedItems.forEach((sound) => {
+        vm.addSound(sound, target.id);
+      });
+    }
+    deletedItems = [];
+  }
+
+  addon.tab.createEditorContextMenu(
+    async (ctx) => {
+      if (
+        await addon.tab.confirm(
+          msg(ctx.type === "costume" ? "confirmCostume" : "confirmSound"),
+          msg(ctx.type === "costume" ? "infoCostume" : "infoSound")
+        )
```

Comment:
```suggestion
        await addon.tab.confirm(
          msg(ctx.type === "costume" ? "confirmCostume" : "confirmSound"),
          msg(ctx.type === "costume" ? "infoCostume" : "infoSound"),
          { useEditorClasses: true }
        )
```
This will make the popup more similar to other popups in the editor and affected by editor dark mode. Editor popups are smaller, so the title is too long - you could change it to "Delete Others" and move the current title to the description.